cppapi/server/tango.h

```diff
@@ -45,6 +45,17 @@
 
 #include <idl/tango.h>
 
+//
+// A short inline function to hide the CORBA::string_dup function
+//
+namespace Tango {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this namespace to match the regular expression: [a-z][a-z_0-9]*+ ([cpp:NamespaceName](https://sonarcloud.io/organizations/tango-controls/rules#rule_key=cpp%3ANamespaceName))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.tango-controls%3Acpp-tango%3Atango-9-lts&pullRequest=474&issues=AWWureZx_eEZPtUmFiG4&open=AWWureZx_eEZPtUmFiG4)

[ISSUE_KEY]: AWWureZx_eEZPtUmFiG4